<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataBase</class>
 <widget class="QMainWindow" name="DataBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>601</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1067</width>
    <height>601</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1067</width>
    <height>601</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DataBase</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1067</width>
     <height>555</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1067</width>
     <height>555</height>
    </size>
   </property>
   <widget class="QPushButton" name="open_DB_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>851</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_db_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>211</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="table_comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>291</width>
      <height>26</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="bd_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>291</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите имя БД (по умолчанию university)</string>
    </property>
   </widget>
   <widget class="QLabel" name="table_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>311</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите таблицу (по умолчанию univer)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="del_id">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="del">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Удаление</string>
    </property>
   </widget>
   <widget class="QLabel" name="del_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Введите id</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>326</x>
      <y>0</y>
      <width>20</width>
      <height>191</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="del_but">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>456</x>
      <y>0</y>
      <width>20</width>
      <height>191</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="add">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавление</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="add_id">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>50</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="add_hours">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>50</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="add_exp">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="add_stavka">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>50</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_but">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>150</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QLabel" name="add_label0">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>id</string>
    </property>
   </widget>
   <widget class="QLabel" name="add_label1">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>часы</string>
    </property>
   </widget>
   <widget class="QLabel" name="add_label2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>опыт</string>
    </property>
   </widget>
   <widget class="QLabel" name="add_label3">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ставка</string>
    </property>
   </widget>
   <widget class="QLabel" name="add_label4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>профессор</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="add_professor">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>100</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="add_label5">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>80</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>кафедра</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="add_kafedra">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>100</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="add_label6">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>предмет</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="add_subject">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>150</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="management">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Управление</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>0</y>
      <width>21</width>
      <height>191</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>170</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="filt_hours30">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>40</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>часы &gt;= 30</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filt_hours50">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>90</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>часы &lt; 50</string>
    </property>
   </widget>
   <widget class="QLabel" name="filt">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>0</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Фильтрация</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filt_stop">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>140</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Сброс</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sort_stop">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>360</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Сброс</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sort_ASC">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>290</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/arrow_up.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="sort_DESC">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>290</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/arrow_down.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sort">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>200</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортировка</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sort_comboBox">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>240</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>id</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>id</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>профессор</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>кафедра</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>предмет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>часы</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>опыт</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ставка</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1067</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
